### ISSUE TITLE
no steal in steal-tools/node_modules

### ISSUE THREAD

**Issue by Alex0007** [stealjs/steal-tools#202]
When i install steal-tools into project, there is no *steal* folder in `myproject/node_modules/steal-tools/node_modules/` so i need to run `npm i` inside steal-tools folder

OS: Linux
npm: 2.7.1

**Comment by matthewp** [comment]
I'm not understanding, why does there need to be a steal folder there?

**Comment by Alex0007** [comment]
Coz it is specified in [dependencies](https://github.com/stealjs/steal-tools/blob/master/package.json#L17)

**Comment by matthewp** [comment]
What I'm asking is, what breaks by steal not being there?  npm doesn't always install dependencies in a project's own node_modules.

**Comment by Alex0007** [comment]
Build task fails without that folder

```
Transpiling...
Minifying...
Calculating main bundle(s)...
Flattening main bundle(s)...
ERROR: ENOENT: no such file or directory, open '/home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/node_modules/steal/steal.production.js' Error: ENOENT: no such file or directory, open '/home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/node_modules/steal/steal.production.js'
    at Error (native)
    at Object.fs.openSync (fs.js:546:18)
    at Object.fs.readFileSync (fs.js:396:15)
    at module.exports (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/node/make_steal_node.js:8:15)
    at module.exports (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/bundle/add_steal.js:8:129)
    at /home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/build/multi.js:205:5
    at Array.forEach (native)
    at /home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/build/multi.js:196:17
    at tryCatchReject (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1183:30)
    at runContinuation1 (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1142:4)
    at Fulfilled.when (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:930:4)
    at Pending.run (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:821:13)
    at Scheduler._drain (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:97:19)
Potentially unhandled rejection [2] Error: ENOENT: no such file or directory, open '/home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/node_modules/steal/steal.production.js'
    at Error (native)
    at Object.fs.openSync (fs.js:546:18)
    at Object.fs.readFileSync (fs.js:396:15)
    at module.exports (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/node/make_steal_node.js:8:15)
    at module.exports (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/bundle/add_steal.js:8:129)
    at /home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/build/multi.js:205:5
    at Array.forEach (native)
    at /home/alex0007/Development/projects/bbooks-manager/node_modules/steal-tools/lib/build/multi.js:196:17
    at tryCatchReject (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1183:30)
    at runContinuation1 (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1142:4)
    at Fulfilled.when (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:930:4)
    at Pending.run (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:821:13)
    at Scheduler._drain (/home/alex0007/Development/projects/bbooks-manager/node_modules/steal/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:97:19)
```

**Comment by daffl** [comment]
I think this is a duplicate of #193

**Comment by matthewp** [comment]
Yes, it might be @Alex0007 can you post your build script or gruntfile or whatever?

**Comment by Alex0007** [comment]
```
    stealTools.build({
        main: 'app',
        bundlesPath: '../public/assets',
        config: __dirname + '/package.json!npm'
    }, {
        bundleSteal: true
    })
```

**Comment by matthewp** [comment]
Yes this is a bug involving `bundleSteal`, same as #193. Thanks for reporting, will try to get this fixed soon.

**Comment by matthewp** [comment]
Fixed in 0.7.4
